INSTALL-LINUX.md

```diff
@@ -172,3 +172,9 @@ On multiple distributions, there are Sonic Pi releases that you can install from
       This sets the CMAKE_BUILD_TYPE option when generating the build system files.
 
       It must be followed by one of the following configurations: `Release, Debug, RelWithDebInfo, or MinSizeRel`
+
+## Troubleshooting 
+### Booting error
+* Is Jack server is running?
```

Comment:
s/Is Jack server is running?/Is Jack server running?

Reply:
I don't understand your comment, @nicoder.
Is Jack not necessary for Linux? 
I only managed to run sonic pi with Jack.  Can you show me how not to do it?
Thanks

Reply:
@tomyo He means that you have a typo, and suggest remove `is` from your sentence.